payments-core/src/main/java/com/stripe/android/payments/PaymentFlowFailureMessageFactory.kt

```diff
@@ -1,22 +1,30 @@
 package com.stripe.android.payments
 
 import android.content.Context
+import androidx.annotation.RestrictTo
 import com.stripe.android.R
 import com.stripe.android.StripeIntentResult
 import com.stripe.android.model.PaymentIntent
 import com.stripe.android.model.SetupIntent
 import com.stripe.android.model.StripeIntent
 
-internal class PaymentFlowFailureMessageFactory(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Is there another way we can keep this internal?

Reply:
The other options that I know of are (from most painless to more painful):

1. Open this class to the library (what I did)
2. Move the class to a shared package IE com.stripe.android
3. Copy the bits I need out of the class and make my own within the scope of PaymentSheet.
4. Change up the payment result class to give a message if it's an error. (not sure if this one is possible, or desirable since we want to give SDK users the option to make their own strings right?)

Reply:
I think we could avoid this change, please refer to the comments to directly mock the class

Reply:
fixed here: [51992c5](https://github.com/stripe/stripe-android/pull/4159/commits/51992c5164de08c278682c50262c5d5244869f1f)